app/src/main/java/com/imobile3/groovypayments/ui/main/MainDashboardActivity.java

```diff
@@ -102,7 +104,17 @@ private void handleDashboardButtonClick(@NonNull MainDashboardButton button) {
                 break;
 
             case Placeholder2:
-                throw new RuntimeException("User clicked a Placeholder button");
+                //throw new RuntimeException("User clicked a Placeholder button");
+                AlertDialog.Builder builder = new AlertDialog.Builder(MainDashboardActivity.this);
```

Comment:
You don't need to create a new AlertDialog, there is already helper method and string xml resources for the strings